docs/advanced/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: index
+title: Advanced
+navigation_weight: 1
+---
+
+# Caching CDN Requests or Non-200 Responses
+
+If an attempt is made to cache an asset that is on a different origin or
+the asset returns a non-200 response, these modules will throw an error. The
+reason for this default behavior is to ensure that only good and local
+responses are cached.
+
+Depending on how you are using these libraries will depend on how you can add
+support for external resources.
+
+## sw-lib
+
+In sw-lib, you can set up some additional options with a route that will allow
+support for additional status codes and / or based on specific headers.
+
+```javascript
+const cdnCacheStrategy = goog.swlib.staleWhileRevalidate({
+  cacheableResponse: {
+    statuses: [0],
+  },
+});
+goog.swlib.router.registerRoute(/http://cdn.mysite.com/styles/.*/, cdnCacheStrategy);
```

Comment:
Could you switch this example to use `new RegExp('^https://cdn.mysite.com/styles/')`

- It uses `https` instead of `http`.
- It uses a `RegExp(string)`-style constructor, avoiding the need to escape the `/` characters inside of a RegExp literal.
- It ensures that the match is done against the beginning of the URL string, which is where we expect the `https://...` to appear.
- It omits the `.*`, as it's not necessary. It would only be needed if there was a trailing `$` character in the RegExp. As long as the RegExp matches, even if it doesn't match the entire URL string, it will trigger the route.

(This tells me that the docs around RegExp-based routes should be improved. I had a similar conversation with @jpmedley earlier.)